test/cpp/interop/xds_interop_server.cc

```diff
@@ -65,13 +66,59 @@ using grpc::testing::SimpleResponse;
 using grpc::testing::TestService;
 using grpc::testing::XdsUpdateHealthService;
 
+namespace {
+constexpr absl::string_view kRpcBehaviorMetadataKey = "rpc-behavior";
+constexpr absl::string_view kErrorCodeRpcBehavior = "error-code-";
+
+std::set<std::string> GetRpcBehaviorMetadata(ServerContext* context) {
+  std::set<std::string> rpc_behaviors;
+  auto rpc_behavior_metadata =
+      context->client_metadata().equal_range(grpc::string_ref(
+          kRpcBehaviorMetadataKey.data(), kRpcBehaviorMetadataKey.length()));
+  for (auto metadata = rpc_behavior_metadata.first;
+       metadata != rpc_behavior_metadata.second; metadata++) {
```

Comment:
prefer ++metadata over metadata++

Reply:
Done. Should we add a clang-tidy check for this?